src/core/shape/attributes.js

```diff
@@ -172,27 +301,76 @@ p5.prototype.rectMode = function(m) {
 };
 
 /**
- * Draws all geometry with smooth (anti-aliased) edges. <a href="#/p5/smooth">smooth()</a> will also
- * improve image quality of resized images.
+ * Draws certain features with smooth (antialiased) edges.
```

Comment:
similar comment as 'noSmooth'
Could you share the reason to make the change from 'Draws all geometry' to 'Draws certain features'? It seems make more sense to 'draw geometry' than 'draw features'. If it's important to keep 'certain', how about 'Draws certain geometry'? @nickmcintyre